src/fmu/ensemble/ensemblecombination.py

```diff
@@ -1,9 +1,11 @@
-"""Module for handling linear combinations of ensembles. """
+"""Module for handling linear combinations of ensembles"""
 
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
+import fnmatch
+import six
```

Comment:
fnmatch from standard library, six not (I think 😉)
```suggestion
import fnmatch

import six
```

Reply:
fixed